### ISSUE TITLE
Using with Parcel

### ISSUE THREAD

**Issue by bubnenkoff** [visjs/vis-network#917]
I am using Vue with vis-network with integration plugin https://github.com/r3code/vue-vis-network/
I am using Parcel instead of webpack.

When I am trying to run simple project I am getting error:
https://user-images.githubusercontent.com/6514326/76624917-443aa980-6547-11ea-9af8-b3f52d1bf57f.png
![изображение](https://user-images.githubusercontent.com/6514326/89279205-c55a2080-d64f-11ea-9430-9e30928895d0.png)

I think it's issue of vis-network

**Comment by Thomaash** [comment]
Hi @bubnenkoff,

this looks like your toolchain doesn't understand ESM (I have never used Parcel though). In your situation, I would try the UMD build (`node_modules/vis-network/standalone/umd/vis-network.js`) to see whether it's ESM/UMD issue or something else.

If you really think there's something wrong with Vis Network, you have to say what and provide an MWE, because the error comes from a file called `vueVisNetwork.common.js` and we don't release anything like that (our filenames are in kebab-case).

**Comment by bubnenkoff** [comment]
You are suggesting to replace file with this `node_modules/vis-network/standalone/umd/vis-network.js`?

**Comment by Thomaash** [comment]
I'm sorry, I don't understand what you mean, could you rephrase it? Thanks.

PS: You didn't really provide anything useful, so it's perfectly possible that what I typed is completely off and doesn't make any sense in your situation.

**Comment by bubnenkoff** [comment]
Sorry. I am not from JS world, I just need to get work one simple project.

I thought you are suggesting me to replace one lib to another and see what's will happens. 

Btw Parcel 1 do not support ESM, but Parcel 2 is very buggy. So is there any way to use not ESM build? If yes, how to do it?

**Comment by Thomaash** [comment]
If you were using Vis Network directly, it would be simple, just import directly `vis-network/standalone/umd/vis-network.js` from your code and the ESM issues would disappear. However since your using r3code's solution, you'll have to ask them for UMD build.

I'm closing this as the issue is not caused by this project.
